problems/3.conditionals.js

```diff
@@ -2,15 +2,23 @@
 * @param {number} temp - The current temperature
 * @returns {string} - A message about the temperature
 */
-function temperatureMessage() {
-
+const  temperatureMessage = (temp) => {
+  if (temperature < 32){
+    return " It's freezing!"
+  }else if (temperature > 32){
+    return "It's alright!"
+  }else {
+    return "It's warm"
+  }
```

Comment:
This is pretty good. But make sure to run the tests. It was not looking for an it's warm option. 
Also will say temperature is not defined. That's b/c you've called your parameter temp.